less/udata/auth.less

```diff
@@ -0,0 +1,42 @@
+.auth-container {
+    form {
+        background-color: white;
+        border: 1px solid darken(@gray-lighter, 10%);
+        border-radius: 4px;
+        padding: 10px;
+        margin-bottom: 20px;
+    }
+}
+
+.social-providers-list {
+    a:hover, a:focus {
+        text-decoration: none;;
+    }
+
+    img {
+        max-width: 64px;
+        margin: 5px;
+
+        &:hover, &:focus {
+
+        }
+    }
+
+    .twitter-logo {
+        display: inline-block;
+        background-color: #55ACEE; // Official color from twitter guidelines
```

Comment:
The file is copy-pasted from Youckan, I don't think this is actually used anywhere; if there are no plans to reintroduce the social network login in the future, it could be removed.